src/main/java/org/gridsuite/study/server/service/RootNetworkNodeInfoService.java

```diff
@@ -254,7 +252,6 @@ public InvalidateNodeInfos invalidateRootNetworkNode(RootNetworkNodeInfoEntity r
         InvalidateNodeInfos invalidateNodeInfos = getInvalidationComputationInfos(rootNetworkNodeInfoEntity, invalidateTreeParameters.computationsInvalidationMode());
 
         if (notOnlyChildrenBuildStatus) {
-            rootNetworkNodeInfoEntity.getModificationReports().forEach((key, value) -> invalidateNodeInfos.addReportUuid(value));
```

Comment:
That's kind of a regression, now we don't delete reports when we invalidate nodes. That's what we wanr in some cases but not for all cases. Maybe we should try to be more precise here. When no children are built we want to delete all the reports upward the next built node.